lib/src/main/ktx/com/fabernovel/statefullayout/StatefulLayout.kt

```diff
@@ -75,7 +75,7 @@ fun StatefulLayout.showError(
 ): State {
     return showState(R.id.stateError).apply {
         if (errorMessage != null) setErrorMessage(errorMessage)
-        if (onRetryListener != null) setErrorRetryListener(onRetryListener)
+        setErrorRetryListener(onRetryListener)
```

Comment:
BEFORE : You can override the error layout without providing a retry button AND pass a null `onRetryListener` by just calling the extension method `showError("error message")`

AFTER : If you do this, the `requireNotNull(retryButton)` in `private fun State.setErrorRetryListener` will be triggered and the app will crash

-> in `setErrorRetryListener` you need to check the requireNotNull only if `onRetryListener` is not null